Userland/Libraries/LibJS/Parser.cpp

```diff
@@ -365,7 +365,8 @@ NonnullRefPtr<Declaration> Parser::parse_declaration()
 NonnullRefPtr<Statement> Parser::parse_statement(AllowLabelledFunction allow_labelled_function)
```

Comment:
This commit got kind of big but I didn't want to litter with a bunch of tiny commits. I can split them up if needed